internal/pkg/cmd/prerunner.go

```diff
@@ -168,8 +168,7 @@ func (c *AuthenticatedCLICommand) AuthToken() string {
 
 func (c *AuthenticatedCLICommand) EnvironmentId() string {
 	if c.Context.GetEnvironment() == nil {
-		noEnvSuggestions := errors.ComposeSuggestionsMessage("This issue may occur if this user has no valid role bindings. Contact an Organization Admin to create a role binding for this user.")
-		output.ErrPrintln(errors.EnvNotSetErrorMsg + noEnvSuggestions)
+		output.ErrPrintln(errors.EnvNotSetErrorMsg + errors.ComposeSuggestionsMessage(errors.EnvNotSetSuggestions))
```

Comment:
Let's prefix this with `WARNING:` for now, and then in the future when we display warnings by default, we can convert this to a real warning.

Reply:
We could, but it would end up being more verbose I think since we're not actually returning an error here. So we would have to print the error message and then also call `DisplaySuggestionsMessage`.

Reply:
Can't we do something like the following?
1. `err := NewErrorWithSuggestions()`
2. `output.Println(err)`

Reply:
Just tried
```
err := errors.NewErrorWithSuggestions(errors.NoEnvironmentFoundErrorMsg, errors.NoEnvironmentFoundSuggestions)
output.ErrPrintln(err)
```
and the suggestion doesn't print:
```
no environment found
```
Same result if I do `output.ErrPrintln(err.Error())`.

Since the suggestion is stored in the `ErrorWithSuggestionImpl` struct I also tried casting to (a pointer to) this struct but it still only prints the error: `output.ErrPrintln(err.(*errors.ErrorWithSuggestionsImpl))`.